### ISSUE TITLE
UNSTABLE Build Linux Wheels / pytorch/executorch / upload / manywheel-py3_8-cpu

### ISSUE THREAD

**Issue by huydhn** [pytorch/executorch#3987]
ET could not be build with python 3.8 atm.  Creating this issue to hide the failure on CI until 3.8 build could be included/excluded properly.

Related issues:
* https://github.com/pytorch/executorch/issues/3988

**Comment by guangy10** [comment]
@huydhn @dbort I don't think there is a plan to make it work for py3.8. I think we are aiming to make it work for higher version e.g. py3.12 instead. Given that, there will be no need to fix these py3_8 jobs. Is there a way to completed disable/remove it from CI instead?

**Comment by huydhn** [comment]
Yes, I have a task to make the list of python versions configurable so that ET could choose which the version it needs.   Then we can remove 3.8 from ET build.

**Comment by huydhn** [comment]
This is not needed anymore after https://github.com/pytorch/executorch/pull/7064
